### DISCUSSION TITLE
Is there a way to install and run podman on ubuntu 14.04 ?

### DISCUSSION THREAD

**Discussion by rishabhj1717** [containers/podman#14244] [Q&A]
I wanted to install podman on ubuntu 14.04 and was following this document https://phoenixnap.com/kb/install-podman-on-ubuntu, but it seems to fail with the following error, when I try to add the apt-key
```
wget -nv https://download.opensuse.org/repositories/devel:kubic:libc│
ontainers:stable/xUbuntu_${VERSION_ID}/Release.key -O- | sudo apt-key add -                                    │
ERROR: cannot verify download.opensuse.org's certificate, issued by '/C=US/O=Let\'s Encrypt/CN=R3':            │
  Issued certificate has expired.                                                                              │
To connect to download.opensuse.org insecurely, use `--no-check-certificate'.                                  │
gpg: no valid OpenPGP data found.
```

Is there a way to install podman on ubuntu 14.04, and if yes, could you please share the steps?

**Comment by vrothberg** [comment]
Thank you for reaching out. Ubuntu 14:04 is too old.  The Kubic repositories are discontinued since Ubuntu ships with Podman in its main repositorie now (see https://podman.io/getting-started/installation#linux-distributions).

**Reply by rishabhj1717** [reply]
Thank you 👍

**Comment by flouthoc** [comment]
@rishabhj1717 If your distro is too old and in case if you are unable to find any other means of installing podman on your distro I'd also like to recommend building podman from source as last option and running it https://podman.io/getting-started/installation#building-from-scratch, works for me most of time.

**Reply by rishabhj1717** [reply]
Thank you, shall try this 👍
